src/Symfony/Component/Validator/Mapping/Factory/MetadataFactoryInterface.php

```diff
@@ -10,8 +10,7 @@
  */
 
 namespace Symfony\Component\Validator\Mapping\Factory;
-
```

Comment:
Thanks. This can be probably checked by coding standard validator. Can I add it somewhere?